src/products/dto/create-product.dto.ts

```
@@ -0,0 +1,72 @@
+
+  @IsOptional()
+  @IsArray()
+  @IsString({ each: true })
+  images?: string[];
+
+  @IsNumber()
+  @Min(0)
+  @Transform(({ value }) => parseInt(value))
+  stockQuantity: number;
+
+  @IsOptional()
```

ESLint

[error] Delete `␍`

(prettier/prettier)